src/main/java/com/icthh/xm/ms/entity/util/CustomCollectionUtils.java

```diff
@@ -1,14 +1,9 @@
 package com.icthh.xm.ms.entity.util;
 
+import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
 import lombok.experimental.UtilityClass;
-import org.apache.commons.collections4.MapUtils;
-import org.apache.commons.lang3.StringUtils;
-
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
please fix according to playbook and code style

Reply:
What do you mean? Have no idea about project **playbook** and **code style**

Reply:
i have send link to playbook in slack. 
Please fix imports according to code style